src/modules/eventListeners.js

```diff
@@ -0,0 +1,93 @@
+import {
+  addListContainer, list, clear, warning,
+} from './variables.js';
+import Tasks from './class.js';
+
+const tasks = new Tasks();
+
+export const newActivity = addListContainer.addEventListener('keyup', (e) => {
+  if (e.keyCode === 13) {
+    if (e.target.value === '') {
+      warning.innerHTML = 'Please enter a task';
+      warning.classList.remove('hide');
+      setTimeout(() => {
+        warning.innerHTML = '';
+        warning.classList.add('hide');
+      }, 2000);
+    } else {
+      const newValue = e.target.value;
+      tasks.addRecord(newValue, false, tasks.activities.length + 1);
+      tasks.local();
+      e.target.value = '';
+    }
+  }
+});
+
+export const deleteActivity = list.addEventListener('click', (e) => {
+  if (e.target.tagName === 'P') {
+    e.path[1].childNodes[5].classList.add('hide');
+    e.path[1].childNodes[7].classList.remove('hide');
+  } else if (e.target.innerHTML === 'delete') {
+    const value = e.path[2].childNodes[3].innerHTML;
+    tasks.eliminate(value);
+  }
+});
```

Comment:
**You made a great effort on this project! It's working perfectly fine! 👏👏👏**

However, you also did the next project's activities such as delete action... In that case, you must put a link to the additional code in the Pull Request created for the next project. For example, you should create a link(s) to specific commit(s) and/or line(s) of code in Pull Request for Milestone 3 and include that link in the description of Pull Request for Milestone 4.